pom.xml

```diff
@@ -120,7 +120,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>google-oauth-plugin</artifactId>
-      <version>0.7</version>
+      <version>0.9-SNAPSHOT</version>
```

Comment:
This needs to be changed before merging, but this version is used for testing and building locally.